src/main/java/hr/eestec_zg/populator/Populator.java

```diff
@@ -25,21 +31,82 @@ public static void main(String[] args) {
         AnnotationConfigApplicationContext ctx = new AnnotationConfigApplicationContext(
                 AppConfig.class, CoreConfig.class);
 
+        //-----------------------------------------------------------------------------
```

Comment:
Makni nepotrebne linije ili komentare poput ovoga.

Reply:
Što nije preglednije tako? :)

Reply:
nece bit potrebe za tim nakon ovog refaktoriranja, vidit ces ;)

Reply:
Da šefe :P